### ISSUE TITLE
Add hyperlinks to cells so entire link doesn't have to be displayed

### ISSUE THREAD

**Issue by cole-at-quadratic** [quadratichq/quadratic#2126]
What I'd love to be able to do:

<img width="359" alt="Screenshot 2024-12-11 at 5 30 15 PM" src="https://github.com/user-attachments/assets/0bbf140c-737e-4bef-a5fb-a67390207485" />

<img width="356" alt="Screenshot 2024-12-11 at 5 30 59 PM" src="https://github.com/user-attachments/assets/f18ae38e-078d-420e-8723-4e7ae9f3c3f7" />

But it is not possible. This would be useful for knowledge workers.

**Comment by davidfig** [comment]
Do you mean optionally replacing the link with text, but if you click it, it still works?

**Comment by cole-at-quadratic** [comment]
@davidfig yes but a user should be able to add a hyperlink using CMD+K on mac to any part of the text, as shown here. Still makes the whole cell clickable, but allows you to have more information
![CleanShot 2024-12-20 at 08 59 32@2x](https://github.com/user-attachments/assets/81b2a7dc-5d8c-4e70-a5e6-9b004f6953c2)
